packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -55,7 +56,12 @@ const styles = (theme: ThemeType): JssStyles => ({
     flexWrap: "wrap",
     color: theme.palette.lwTertiary.main,
     marginTop: 8,
-    ...separatorBulletStyles(theme)
+    [theme.breakpoints.up('sm')]: {
+      ...separatorBulletStyles(theme, 2),
```

Comment:
Since the second parameter you added to `separatorBulletStyles` is optional with a default value of 2, you don't need to explicitly pass in 2

Reply:
I know, but I think it looks a little clearer this way? That way someone reading the code have a sense of the relative difference between the two (I recognize this doesn't do much without the name of the parameter)